ufunc coverage changed in numpy dev

Example log: https://travis-ci.org/astropy/astropy/jobs/316592597

```
_______________________ TestUfuncCoverage.test_coverage ________________________
self = <astropy.units.tests.test_quantity_ufuncs.TestUfuncCoverage object at 0x7f796e3cd470>
    def test_coverage(self):
        all_np_ufuncs = set([ufunc for ufunc in np.core.umath.__dict__.values()
                             if type(ufunc) == np.ufunc])

        from .. import quantity_helper as qh

        all_q_ufuncs = (qh.UNSUPPORTED_UFUNCS |
                        set(qh.UFUNC_HELPERS.keys()))

>       assert all_np_ufuncs - all_q_ufuncs == set([])
E       AssertionError: assert {<ufunc 'gcd'>, <ufunc 'lcm'>} == set()
E         Extra items in the left set:
E         <ufunc 'gcd'>
E         <ufunc 'lcm'>
E         Use -v to get the full diff
```

--- a/astropy/units/quantity_helper.py
+++ b/astropy/units/quantity_helper.py
@@ -44,8 +44,11 @@ UNSUPPORTED_UFUNCS = set([np.bitwise_and, np.bitwise_or,
                           np.bitwise_xor, np.invert, np.left_shift,
                           np.right_shift, np.logical_and, np.logical_or,
                           np.logical_xor, np.logical_not])
-if isinstance(getattr(np, 'isnat', None), np.ufunc):
-    UNSUPPORTED_UFUNCS |= {np.isnat}
+for name in 'isnat', 'gcd', 'lcm':
+    # isnat was introduced in numpy 1.14, gcd+lcm in 1.15
+    ufunc = getattr(np, name, None)
+    if isinstance(ufunc, np.ufunc):
+        UNSUPPORTED_UFUNCS |= {ufunc}
 
 # SINGLE ARGUMENT UFUNCS
 

